code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -441,6 +441,15 @@
 	icon_open = "trenchcoat_grey_open"
 	icon_closed = "trenchcoat_grey"
 
+/obj/item/clothing/suit/storage/toggle/trench/greentrench
```

Comment:
Rename it to 
```suggestion
/obj/item/clothing/suit/storage/toggle/trench/green
```
The "trench" exists already as part of the path
Dont forget to change the path of the loadout item aswwell